spring-rabbit/src/main/java/org/springframework/amqp/rabbit/annotation/RabbitListenerAnnotationBeanPostProcessor.java

```diff
@@ -236,21 +239,55 @@ public void doWith(Method method) throws IllegalArgumentException, IllegalAccess
 				}
 			}
 		});
-		if (classLevelListener != null) {
-			processMultiMethodListener(classLevelListener, multiMethods, bean, beanName);
+		if (hasClassLevelListeners) {
+			processMultiMethodListeners(classLevelListeners, multiMethods, bean, beanName);
 		}
 		return bean;
 	}
 
-	private void processMultiMethodListener(RabbitListener classLevelListener, List<Method> multiMethods, Object bean,
-			String beanName) {
+	/*
+	 * AnnotationUtils.getRepeatableAnnotations does not look at interfaces
```

Comment:
I didn't open a JIRA issue - the `JmsListener` used `getAttribute()` not `findAttribute()` before this change. So it's not an issue for them (I guess JMS doesn't support `JmsListener` on interfaces).

I guess we should bring it to Stephane and/or Juergen's attention.